### PR TITLE
Adds in an option for admins to save logs to disk

### PR DESCRIPTION
**Pull request by BobdaBiscuit**
Adds in an alert box to .getruntimelog, .getserverlog, Show Server Log and Show Server Attack Log. You can choose to open the file in notepad (as in current behaviour) or save it to disk.

Thought behind this is us on linux can't use the current verb and multiple admins on windows are reporting the current verb does't work either on occasionas and it's 99% gonna be byond shitting the bed with notepad. If this pr still doesn't work we can most likely lynch pomf or something.

### CODE REVIEW

**Comment by clusterfack** [comment]
Wow talk about flubbed

**Comment by BobdaBiscuit** [comment]
pls no bully

**Review by PJB3005 — APPROVED** [review]

File: code/modules/admin/verbs/getlogs.dm [hunk 1]
```diff
@@ -57,10 +57,18 @@
 		#warn Run is deprecated and disabled for some fucking reason in 507.1275/6, if you have a version that doesn't have run() disabled then comment out #define RUNWARNING in setup.dm
 	src << ftp(file(path))
 	#else
-	src << run(file(path))
+	var/method = alert("What method do you wish to get the logs?",,"Save to file","Open in notepad")
```
Comment (ComicIronic) [inline code/modules/admin/verbs/getlogs.dm]: Replace all this copypaste with a PROC

**Review by ComicIronic — CHANGES_REQUESTED** [review]
DO THAT

**Comment by BobdaBiscuit** [comment]
No doubt I did something terribly wrong, it just werks(tm).

**Pull request merged**